### PR TITLE
remove features covered by the new release of official plugin

### PR DESCRIPTION
**Pull request by giovannitoledols**

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -150,11 +138,8 @@ export class OpsLevelGraphqlAPI implements OpsLevelApi {
         framework = entity.metadata.tags?.find(tag => this.frameworks.includes(tag));
       }
 
-      tierAlias = entity.metadata.annotations?.["opslevel.com/tier"]
-
       response = this.client.request(serviceUpdate, { alias: entityAlias,
                                                       language: primaryLanguage?.name,
-                                                      tierAlias: tierAlias,
```
Comment (esnunes) [inline src/api.ts:157]: Should we remove the tier definition?

Comment (giovannitoledols) [reply src/api.ts:157]: new mutation on ops level side already gets this.

Comment (esnunes) [reply src/api.ts:157]: cool, is it expecting you to set the tier using the `opslevel.com/tier` annotation too? Or do you have to set it somewhere else?

Comment (giovannitoledols) [reply src/api.ts:157]: the same as before, with the `opslevel.com/tier` annotation

Comment (giovannitoledols) [reply src/api.ts:157]: good thing is that the repository inherits the tier information from the service

**Review by raphapr — APPROVED** [review]

**Pull request merged**